build.gradle

```diff
@@ -140,57 +153,43 @@ subprojects {
     }
 }
 
-publishing {
-    publications {
-        maven(MavenPublication) {
+bintrayUpload.dependsOn build
+
+task aggregateJavadoc(type: Javadoc,
+    description: "Generate javadocs from all child projects as if it was a single project",
+    group: JavaBasePlugin.DOCUMENTATION_GROUP
+) {
+    destinationDir file("${rootDir}/build/docs/aggregateJavadoc")
```

Comment:
Using `rootDir` you will have make sure your javadoc are be aggregate in one place is save, and will not. Executing task in subproject will make sure if it is all in one place. It is for safe.

Reply:
The build folder must be the target for everything we generate. Using rootDir means creating a new folder named docs at the same level than "src" and all subprojects.

We don't expect any clash because we only want to run this task at the root. This is not a subproject task.

Reply:
I've better way `rootProject.buildDir`